packages/react/src/components/Chat/ChatMessage.tsx

```diff
@@ -138,13 +141,14 @@ class ChatMessage extends UIComponent<WithAsProp<ChatMessageProps>, ChatMessageS
     reactionGroupPosition: 'start',
   }
 
+  updateActionsMenuPosition = () => {}
+
   state = {
     focused: false,
     isFromKeyboard: false,
+    messageDomNode: null,
```

Comment:
it is necessary to use `state` to store this DOM node, as otherwise `Popper` component that is used for `actionsMenu` rendering won't update its target once `messageDomNode` is set